src/app/reducers/interactives.js

```diff
@@ -43,14 +42,14 @@ export default function reducer(state = initialState, action = {}) {
             return Object.assign({}, state, {
                 errors: initialState.errors,
             });
-        case types.FILTER_INTERACTIVES: {
-            const { topics, query } = action.filters;
+        case types.SORT_INTERACTIVES: {
+            const { sort } = action;
             let filteredInteractives = state.interactives;
-            if (topics.length > 0) {
-                filteredInteractives = state.interactives.filter(interactive => isInArray(topics, interactive.metadata.primary_topic));
+            if (sort === "title") {
+                filteredInteractives = filteredInteractives.sort((a, b) => a.metadata.label.localeCompare(b.metadata.label));
```

Comment:
Just a suggestion for next time, it helps the reviewer if your args are human readable, thanks.